src/app/container/info-tab/info-tab.component.html

```diff
@@ -267,14 +267,45 @@
           <!-- Public Topic -->
           <li *ngIf="isPublic">
             <span><strong matTooltip="Short description of the tool">Topic</strong>:&nbsp;</span>
-            <span>{{ tool?.topicSelection === TopicSelectionEnum.AUTOMATIC ? tool?.topicAutomatic : tool?.topicManual }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.AUTOMATIC">{{ tool?.topicAutomatic }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.MANUAL">{{ tool?.topicManual }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.AI">
+              {{ tool?.topicAI }}
+              <span
+                class="bubble ai-bubble"
+                matTooltip="This topic sentence was AI generated. For more information, please click the bubble to visit our docs"
+              >
+                <img class="ai-icon" src="/assets/svg/ai-icon.svg" alt="AI generation icon" />
+                <span>AI</span>
+                <img class="ai-icon" src="/assets/svg/help.svg" alt="help icon" />
+              </span>
+            </span>
           </li>
           <!-- Private topic Topic -->
           <ng-container *ngIf="!isPublic">
             <li>
               <span><strong matTooltip="Short description of the tool">Topic Automatic</strong>:&nbsp;</span>
               <span>{{ tool?.topicAutomatic }}</span>
             </li>
+            <li>
+              <span
+                ><strong matTooltip="Short description of the tool, from the GitHub repository description generated with AI"
+                  >AI Generated Topic</strong
+                >:&nbsp;</span
+              >
+              <span data-cy="topicAI-text">{{ tool?.topicAI }}</span>
+              <span
+                *ngIf="tool?.topicAI !== null"
+                data-cy="ai-bubble"
+                class="bubble ai-bubble"
+                onclick="window.open('https://docs.dockstore.org/en/stable/faq.html', 'blank')"
```

Comment:
Should use the constant https://github.com/dockstore/dockstore-ui2/blob/8a1faa120c58863192dfb7c0c2d4e181e0d296a3/src/app/shared/dockstore.model.ts#L90